provider/core/src/lib.rs

```diff
@@ -174,10 +174,14 @@ pub mod prelude {
     pub use crate::any::AsDynProviderAnyMarkerWrap;
     #[cfg(feature = "serde")]
     pub use crate::serde::AsDeserializingBufferProvider;
+
+    pub use crate::yoke;
+    pub use crate::zerofrom;
```

Comment:
Thought: Adding these to the prelude is perhaps something to call out, since `yoke` has not previously been part of the prelude.  I think this is fine but I'm not sure.